auditlog/migrations/0012_merge_20220927_1126.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 4.1.1 on 2022-09-27 09:26
```

Comment:
Why do we have this merge migration here?

Reply:
Between the time I applied the changes to my local branch and the time I opened this PR, there were some changes made to auditlog, that force me to create a merge migration. However, I can delete my initial migration and the merge migratiion and create a new one on top.

Reply:
Please do it